cmd/plugin/builder/command/cli_compile.go

```diff
@@ -303,7 +301,7 @@ func buildPlugin(path, id string) (plugin, error) {
 		p.modPath = ""
 	}
 
-	log.Debugy("plugin", p)
+	log.V(4).Infof("plugin %v", p)
```

Comment:
This log is still showing all the time.  I think we need to set the `threshold` of the logger for the `V(x)` to work.

Reply:
Yeah. I thought the default threshold was set to `3` but it was not. Updated as part of https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/24.
This log will not be set unless the threshold is set to `>3`.